.github/workflows/integration-test.yml

```
@@ -0,0 +1,30 @@
+name : Integration Tests
+on :
+  # allow manual triggering 
+  workflow_dispatch:
+  
+  # for debugging
+  push:
+  pull_request:
+
+  schedule:
+    # run Thursday 4:30 AM UTC
+  - cron: '30 4 * * 4'    
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    
+    container: 
+      image: pecan/depends:R4.1
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v3
+    - name: Run tests
+      run: |
+        for FILE in modules/data.atmosphere/inst/integrationTests/*; do
+          Rscript $FILE;
+        done
```

```suggestion
    - name: Install Packages
      run: make -j1 install
      
    - name: Run tests
      run: |
        for FILE in modules/data.atmosphere/inst/integrationTests/*; do
          Rscript $FILE;
        done
```

Let us try with install only the libraries